src/main/java/edu/suffolk/litlab/efspserver/ecf4/TylerModuleSetup.java

```diff
@@ -216,14 +225,21 @@ public void preSetup() {
               .build();
 
       var r = new Random();
+      ScheduleBuilder codeUpdateSchedule;
+      if (scheduleImmediately) {
+        // Testable version! Updates the codes 20 seconds after launch
+        // We also use this for immediately running the update on ephemeral machines that are
+        // controlled by external cron
+        codeUpdateSchedule = SimpleScheduleBuilder.simpleSchedule().withIntervalInSeconds(20);
+      } else {
```

Comment:
Oh, that's a good idea! I'll make the change to allow both schedules.

Two reasons why it turned off the scheduled update, neither of which is that strong:

1. The original commented out code alternated between the two options, so I thought they were mutually exclusive use cases
2. At the time, my intention was to have the code updater in a daily scheduled machine run. That is a short-lived machine instance that is shut down as soon as the code updater finishes. However, that Fly.io feature didn't feel that mature yet so we decided not to rely on it. The operational model was to keep 1 hot instance to avoid the cold start delay, which meant there was no reason not to keep the existing Quartz scheduled update.